interface-definitions/netns.xml.in

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="netns" owner="${vyos_conf_scripts_dir}/netns.py">
+    <properties>
+      <help>Network namespace</help>
+      <priority>299</priority>
```

Comment:
No constraint required for the netns? We should still add a proper constraint here to not allow super weird names.

Reply:
Done